README.md

```diff
@@ -191,7 +194,27 @@ To control keys and certificates used for TLS, emmy CLI programs use several fla
 * `--cert` which expects the path to server's certificate in PEM format, 
 * `--key` which expects the path to server's private key file.
 
-On the other hand, we can provide `emmy client` with the `--cacert` flag, which expects the certificate of the CA that issued emmy server's certificate (in PEM format). Again, if this flag is omitted, the certificate in `test/testdata` directory is used.
+On the other hand, we can provide `emmy client` with the following flags:
+* `--cacert`, which expects the certificate of the CA that issued emmy server's certificate (in PEM format). Again, if this flag is omitted, the certificate in `test/testdata` directory is used.
+* `--insecure`, which tells the client not to check the CA certificate chain or emmy server's hostname. This option is meant for development purposes only, for instance when we have deployed emmy server with a self-signed certificate to a host whose hostname does not match the hostname specified in the server's certificate (for instance, *localhost*).
+  
+  To give you an example, let's try to run an emmy client against an instance of emmy server that uses the self-signed certificate shipped with this repository. The hostname in the certificate is *localhost*, but the server is deployed on a host other than localhost (for instance, *10.12.13.45*). When we try to contact the server withour the *--insecure* flag, here's what happens:
+
+  ```bash
+  $ emmy client --server 10.12.13.45:7007 schnorr
+
+  2017/09/13 12:48:47 [client] 12:48:47.232 GetConnection ▶ INFO 001 Getting the connection
+  Cannot connect to gRPC server: Could not connect to server 10.12.13.45:7007 (x509: cannot validate certificate for 10.12.13.45 because it doesnt contain any IP SANs)
+  ```
+
+  Now let's include the *--insecure* flag, and the (insecure) connection to the server is now successfully established.
+
+  ```bash
+  $ emmy client --server 10.10.43.45:7007 --insecure schnorr
+
+  2017/09/13 12:49:14 [client] 12:49:14.961 GetConnection ▶ INFO 001 Getting the connection
```

Comment:
Perhaps a WARNING would be in place when `--insecure` is used?

Reply:
Indeed. Done.